lib/api/v1/teams.rb

```diff
@@ -84,6 +84,96 @@ class Teams < Grape::API
               authorize team, :member?
               present team.users, with: API::Entities::Users
             end
+
+            desc "Deletes a member from a team",
+              entity:  API::Entities::TeamMembers,
+              failure: [
+                [401, "Authentication fails."],
+                [403, "Authorization fails."],
+                [404, "Not found."]
+              ]
+
+            params do
+              requires :id, documentation: { desc: "Team id" }
+              requires :member_id, documentation: { desc: "Team member id" }
+            end
+
+            delete ":member_id" do
+              team_user = TeamUser.find_by(id: params[:member_id], team_id: params[:id])
+              raise ActiveRecord::RecordNotFound if team_user.nil?
+              authorize team_user, :destroy?
+
+              destroyed = ::TeamUsers::DestroyService.new(current_user).execute(team_user)
+
+              if destroyed
```

Comment:
IIRC, simply doing the following has a similar effect:

```ruby
unless destroyed
  error!({ "errors" => team_user.errors.full_messages }, 422, header)
end
```

As in, in the positive case, then null will be returned, which is fine.

Reply:
I thought that I had tried that and got an exception but I double checked now and it seems to work. Anyway, thanks! :)

Reply:
I was right, for some reason this fails in the tests but not in the browser itself when I try to remove a member.

```
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN">
<HTML>
  <HEAD><TITLE>Internal Server Error</TITLE></HEAD>
  <BODY>
    <H1>Internal Server Error</H1>
    no implicit conversion of nil into String
    <HR>
    <ADDRESS>
     WEBrick/1.3.1 (Ruby/2.4.2/2017-09-14) at
     127.0.0.1:57911
    </ADDRESS>
  </BODY>
</HTML>
```

No idea why.

Reply:
Ah right, it returns a JSON `null` object, that you should then handle on the JS side. I think this also happens on other endpoints. This is something to consolidate either in #1437 or #1500.

Reply:
But it's an internal server error when it should be a success with a no content response. 
There's nothing to handle on the client-side. 

This only happens when running the tests. If I go to the browser and try to delete a member, it works as expected.

Reply:
@mssola any idea on this? ☝️